pages/form/returning-members.js

```diff
@@ -96,96 +93,86 @@ const CONTINUING_STATUS = [
   "I'm undecided or not continuing",
 ];
 
-const FormHeader = ({ title, marginBottom }) => {
-  const theme = useContext(ThemeContext);
-  return (
-    <SystemComponent
-      fontSize={theme.fontSizes.header3}
-      textAlign="center"
-      mb={marginBottom}
-    >
-      <SystemSpan>
-        <Subtitle>{title}</Subtitle>
-      </SystemSpan>
-    </SystemComponent>
-  );
-};
-
 const ReturningMembersForm = () => {
   const theme = useContext(ThemeContext);
 
+  const dispatch = useDispatch();
+  const router = useRouter();
+  const { user, hydrated } = useSelector(state => state.userState);
```

Comment:
Can you explain what `hydrated` is used for?

Reply:
I believe it's to tell the application whether or not the data has been loaded in @williamhpark, though I may be wrong. Pretty sure it's a similar concept to https://reactjs.org/docs/react-dom.html#hydrate though.

Reply:
Yes exactly. This app has been using `redux-persist` for a long time. Redux Persist is being used to automatically store data from the Redux Store into Local Storage, then load it back into the Redux Store (this happens during page refreshes so that certain data in the Redux Store don't get lost).

`hydrated` will become true when all the data has been loaded back into the Redux Store.

Reply:
Ohh interesting, thanks for explaining